core/src/main/java/org/keycloak/protocol/oidc/representations/OIDCConfigurationRepresentation.java

```diff
@@ -142,6 +142,12 @@
     @JsonProperty("backchannel_logout_session_supported")
     private Boolean backchannelLogoutSessionSupported;
 
+    @JsonProperty("authorization_details_supported")
```

Comment:
According to https://datatracker.ietf.org/doc/html/draft-ietf-oauth-rar#section-10 ,
`authorization_details_supported` is not defined.

`OIDCConfigurationRepresentation` only includes server metadata defined by standard specification.

Instead of using non-standard server metadata `authorization_details_supported`, how about the following?

-  `authorization_details_types_supported` being vacant is equivalent to `authorization_details_supported` being false.
-  `authorization_details_types_supported` being not vacant is equivalent to `authorization_details_supported` being true.

Reply:
I agree with you.